ocean_lib/data_provider/data_service_provider.py

```diff
@@ -350,6 +351,17 @@ def get_url(config):
     def get_api_version():
         return DataServiceProvider._remove_slash(os.getenv(ENV_PROVIDER_API_VERSION, DataServiceProvider.API_VERSION))
 
+    @staticmethod
+    def get_service_endpoints():
+        """
+        Return the service endpoints from the provider URL.
+        """
+        if DataServiceProvider.provider_info is None:
+            config = ConfigProvider.get_config()
+            DataServiceProvider.provider_info = requests.get(config.provider_url)
+            DataServiceProvider.provider_info = DataServiceProvider.provider_info.json()
+        return DataServiceProvider.provider_info['serviceEndpoints']
```

Comment:
please add an empty line before the return statement.
I am also confused about L361 and L362 assigning the same variable in consecutive lines. Can you simplify to `requests.get(config.provider_url).json()`?